README.md

```diff
@@ -46,6 +49,21 @@ However, there are challenges with assembling and managing these tools yourself:
 
 The VMClarity project is focused on unifying detection and management of VM security threats in an agentless manner.
 
+# Demo
+<img src="./img/vmclarity_demo.gif" alt="VMClarity demo" />
+
+# Quick start
+1. Deploy the VMClarity AWS CloudFormation [stack](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?stackName=VMClarity&templateURL=https://s3.eu-west-2.amazonaws.com/vmclarity-v0.4.0/VmClarity.cfn).
```

Comment:
I went pretty out of my way to avoid putting version numbers in the README, if we go this route we'll need to add to the release process a step to update the README link with the release version prior to cutting the tag. Otherwise the README for the tagged version will be out of sync with the tag that has been cut.

I would much rather upload this stack to AWS, and generate this link into the release notes for a specific tag, so then you go to releases and each release has a "Deploy to AWS" and "Deploy to Azure" button. We can provide a link from the readme to `https://github.com/openclarity/vmclarity/releases/latest` in order to point people directly to the latest release.

Also which account/person owns the S3 bucket and process for uploading this?